andela/ah-rambo-backend
# #167619168 Create Sessions Table

#### What does this PR do?
- Create migration for session table
- Add demo seed file

#### Description of Task proposed in this pull request?
-  Add session migration file
- Add Session model
- Add Session seed file

#### How should this be manually tested (Quality Assurance)?
- Check out the branch
-Add the database connection string in .env file
- Run ```npm run db:seed```

#### What are the relevant pivotal tracker stories?
- Story ID: [#167619168](https://www.pivotaltracker.com/story/show/167619168)

#### Any background context you want to add (Operations Impact)?
- Requires database connection string for it to work

#### What I have learned working on this feature:
- Use sequelize cli to generate migrations and seeds

#### Screenshots: [If you made some visual changes to the application please upload screenshots here, or remove this section]
- n/a
